doc/api/Track_Selection/getAvailableAudioTracks.md

```diff
@@ -71,16 +71,16 @@ Each of the objects in the returned array have the following properties:
 
 ### Asking for a specific Period
 
-  - `contentProtections` (`Object|undefined`): Encryption information linked to this
-    Representation.
+- `contentProtections` (`Object|undefined`): Encryption information linked to this
+  Representation.
 
-    If set to an Object, the Representation is known to be encrypted. If unset or set to
-    `undefined` the Representation is either unencrypted or we don't know if it is.
+  If set to an Object, the Representation is known to be encrypted. If unset or set to
+  `undefined` the Representation is either unencrypted or we don't know if it is.
 
-    When set to an object, it may contain the following properties:
+  When set to an object, it may contain the following properties:
 
-    - `keyIds` (`Array.<Uint8Array>|undefined`): Known key ids linked to that
-      Representation.
+  - `keyIds` (`Array.<Uint8Array>|undefined`): Known key ids linked to that
+    Representation.
```

Comment:
Wait... I think this is actually a rebase/conflict issue.

That item should be at the end of the previous chapter

Reply:
Do you move it higher up or can I do it? I also have PR awaiting for this :p

Reply:
I've done it